src/main/res/values/strings.xml

```diff
@@ -14,23 +14,23 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 <resources>
-    <string name="about_preference_title">About OpenTracks</string>
-    <string name="about_description"><i>OpenTracks</i> is a sport tracking application that completely respects your privacy.</string>
-    <string name="about_url">Project web page: <a href="%1$s">%1$s</a></string>
-    <string name="about_version_name">Version name: %1$s</string>
-    <string name="about_version_code">Version code: %1$d</string>
+    <string name="about_preference_title">About</string>
+    <string name="about_description">A sport tracking buddy that respects your privacy.</string>
```

Comment:
doesn't sound right; why buddy?

Reply:
It is easier to identify with people, and in terms of privacy that connection becomes meaningful.
A personified concern is less esoteric than just the concept.

Reply:
Moving this change into a separate PR would be great; might be a nice place for others to give there opinion.
Plus: the fastlane files also needs to be changed.

Reply:
https://github.com/OpenTracksApp/OpenTracks/pull/1032